NAMESPACE

```diff
@@ -34,6 +34,7 @@ export(createIndividualCharacteristics)
 export(createPopulation)
 export(createPopulationCharacteristics)
 export(createSimulationBatch)
+export(enum)

```

Comment:
@PavelBal I am trying this... let see if it creates conflicts with TLF or Reporting Engine